lib/presentation/view/header/context_menu_header_builder.dart

```diff
@@ -40,18 +40,21 @@ class ContextMenuHeaderBuilder {
 
   ContextMenuHeaderBuilder(this._key, this._file);
 
-  ListTile build() {
-    return ListTile(
-      key: _key,
-      leading: _file.fileIcon(),
-      title: Transform(
-        transform: Matrix4.translationValues(-16, -2, 0.0),
-        child: Text(
-          _file.fileName(),
-          maxLines: 1,
-          style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+  Transform build() {
+    return Transform.translate(
+        offset: Offset(0,3.5),
+        child: ListTile(
+          key: _key,
+          leading: Transform.translate(
+              offset: _file.fileIconImagePath() == 'assets/images/ic_folder.svg' ? Offset(0,0) : Offset(0,2),
+              child: _file.fileIcon()
           ),
-        ),
-      trailing: _file.fileSize() == 0 ? null : Text(filesize(_file.fileSize()), style: TextStyle(fontSize: 14, color: AppColor.documentModifiedDateItemTextColor)));
+          title: Text(
+            _file.fileName(),
```

Comment:
How about if `fileName` is too long?

Reply:
I added the ellipsis at the end if it's too long